Content.Server/Communications/CommunicationsConsoleSystem.cs

```diff
@@ -255,10 +257,18 @@ private void OnSelectAlertLevelMessage(EntityUid uid, CommunicationsConsoleCompo
                     return;
                 }
 
+                // User has an id
                 if (_idCardSystem.TryFindIdCard(mob, out var id))
                 {
                     author = $"{id.Comp.FullName} ({CultureInfo.CurrentCulture.TextInfo.ToTitleCase(id.Comp.JobTitle ?? string.Empty)})".Trim();
                 }
+
+                // User does not have an id and is a borg, so use the borg's name
+                if (HasComp<BorgChassisComponent>(mob))
+                {
+                    var meta = _entityManager.GetComponent<MetaDataComponent>(mob);
```

Comment:
Why not TryComp?

Reply:
TryComp gives a warning when getting `MetaDataComponent`
![image](https://github.com/user-attachments/assets/05d2e393-818e-41ce-a6ce-d0b6e91c8242)

Reply:
> TryComp gives a warning when getting `MetaDataComponent` ![image](https://private-user-images.githubusercontent.com/83733158/349349427-05d2e393-818e-41ce-a6ce-d0b6e91c8242.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjExODcxNDQsIm5iZiI6MTcyMTE4Njg0NCwicGF0aCI6Ii84MzczMzE1OC8zNDkzNDk0MjctMDVkMmUzOTMtODE4ZS00MWNlLWE2Y2UtZDBiNmU5MWM4MjQyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzE3VDAzMjcyNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQxOGYyYTM2MTQ1MjE1YTYzZTM4ZGZiYTEwYzgzNGQwYmNkNDFkNGIwNDA1OTAzYzM5YTUzNWYxNzM2NzA3NTMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.uHfitfYKWfz3MS7lO-V0kSStTN__-koVXXeJAJJp4lE)

Horrible warning. What even is a non-generic variant

Reply:
use Name(uid) instead